themes/default/content/blog/update-page-improvements/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Enhanced Pulumi Cloud Stack Update Pages"
```

Comment:
What do you think about something a bit more eye-catching? Like:

```suggestion
title: "Pulumi Cloud Gets Full Historical Views for Resources and Stack Outputs"
```

or something like that.